R/excel_dates.R

```diff
@@ -38,6 +38,7 @@
 #' excel_numeric_to_date(40000.521, include_time = TRUE) # Time is included
 #' excel_numeric_to_date(40000.521, include_time = TRUE,
 #'   round_seconds = FALSE) # Time with fractional seconds is included
+#' @family Date conversion
```

Comment:
Let's put a longer note here like "if your input column(s) have a mix of Excel numeric dates and actual dates, see the more powerful functions `convert_to_date` and `convert_to_datetime`."

Reply:
Good call.  Updated.